presto-main/src/main/java/com/facebook/presto/metadata/DiscoveryNodeManager.java

```diff
@@ -250,10 +251,10 @@ private synchronized void refreshNodesInternal()
     {
         // This is currently a blacklist.
         // TODO: make it a whitelist (a failure-detecting service selector) and maybe build in support for injecting this in airlift
-        Set<ServiceDescriptor> services = serviceSelector.selectAllServices().stream()
+        Set<ServiceDescriptor> services = serviceSelector.selectAllServices()
+                .stream()
```

Comment:
nit: the coding style that we usually follow is to keep .stream in the previous line. grep for other usages.